.github/RELEASE_SCHEDULE.md

```diff
@@ -1,16 +1,29 @@
 # DNN Platform Release Schedule
-To ensure adequate time for release planning by the community, partners, and vendors a specific release process will be followed for all releases.  
+To ensure adequate time for release planning by the community, partners, and vendors, a specific release process will be followed for all releases.  
 
 ## Release Candidates
-For a period of one week (Revision), two weeks (Minor) or three-weeks (Major) before any release, a Release Candidate (RC) version will be made available to the public.  At present these release candidates will be for testing only. After version 10.x efforts will be made to support upgrading from RC to Production releases.
+Releases candidates (RC) are designed to give the community time to adjust their existing environments for any breaking changes and identify any unintended changes. Strong community participation during the RC process will result in more stable releases. 
 
-The goal of these release candidates is to give the community time to adjust their existing environments for any breaking changes, as well as to identify any issues with the changes.  If necessary, changes will be incorporated an additional RC release could be made if significant problems are identified.  If a revised Release Candidate is necessary the Production Release schedule may be impacted.  The exact impact will vary on a case-by-case basis depending on the nature of the issue(s) identified during RC review, however, will be clearly communicated as part off the updated RC release notes.
+### Major Releases (Example 10.0.0)
+Major releases will have an initial RC cycle with a minimum duration of three-weeks from the date of RC release and final release. 
+
+### Minor Releases (Example 9.8.x, or 9.9.x)
+Minor releases will have an initial RC cycle with a minimum duration of two weeks from the date of RC release and final release.
+
+### Revision Releases (Example 9.8.1, or 9.8.2)
+Revision releases will have an initial RC cycle with a minimum duration of one week from the RC release and final release.
+
+### Changes During RC Phase
+If necessary, changes will be incorporated during an RC; if the changes resolve significant issues or introduce risk, an additional RC may be created at the Approvers group's discretion.  If a revised RC is necessary, the Production Release schedule will be impacted; the exact impact will vary on a case-by-case basis depending on the nature of the issue(s) identified during the RC review. However, it will be communicated as part of the updated RC release notes.
 
 ## Production Releases
```

Comment:
```suggestion
## Official Releases
```

Reply:
Would "Final" work for you on this?  That matches the other locations

Reply:
What other locations?  I am not really a fan of "Final", but I can change it if you all think it is the right label.